src/EA.Weee.Web/Areas/Admin/Mappings/ToViewModel/ViewAllNotesMapBase.cs

```diff
@@ -16,12 +18,25 @@ protected ViewAllNotesMapBase(IMapper mapper)
 
```

Comment:
This may be a bit of a pain but this code looks very similar to ListOfNotesViewModelBase, wonder if we can reuse that rather than the ViewAllNotesMapBase class.

Reply:
yes, I think it could be re-used.